db/master-data.cds

```diff
@@ -35,9 +34,9 @@ entity Flight : MasterData {
   // when cuid is added, the to_Airline & to_Connection can be made managed association,
   // furthermore the AirlineID and ConnectionID can be removed,
   // they will be replaced by the generate FKs for to_Airline & to_Connection
-  key AirlineID    : String(3);
+  key AirlineID    : String(3) @cds.collate : false;
   key FlightDate   : Date;
-  key ConnectionID : String(4);
+  key ConnectionID : String(4) @cds.collate : false;
```

Comment:
[unrelated] I think the Order of the key elements is strange. Shouldn't it be (airline, connection, date) ?

Reply:
Classic SFLIGHT indeed has them in the right order.